packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -205,6 +206,9 @@ class WebView extends StatefulWidget {
   /// [WebViewController.evaluateJavascript] can assume this.
   final PageFinishedCallback onPageFinished;
 
+  /// The custom UserAgent.
```

Comment:
We should have more information in this Dartdoc, some question I think I might be having if I were looking here:

1. What would be the user agent when this is null?
2. What happens when the widget is rebuild with a new userAgent? is it sending a new request with the new user agent?
3. What happens when `goBack` is called after a user agent was changed? is the new or previous user agent used?

In general I recommend reading the [documentation section in the Flutter style guide](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#documentation-dartdocs-javadocs-etc) which we should follow here.